site/en/posts/customize-pwa-window-launch.md

```diff
@@ -0,0 +1,49 @@
+  - **Retain** - the most recently interacted with browsing context won’t be overridden but is chosen to handle the launch. A new `LaunchParams` with its `targetURL` set to the launch URL will be enqueued in the document’s [`window.launchQueue`](https://web.dev/launch-handler/#the-window.launchqueue-interface). You can then customize how to handle the launch. If you don’t handle the incoming `LaunchParams`, the app will come into focus but no navigation is made. So make sure to handle the launch appropriately for a good experience for your users.
+
+Note: For these scenarios, browsing context usually means the installed PWA window. However, in the case of tabbed application mode (currently in [developer trial behind a flag](https://web.dev/tabbed-application-mode/)), browsing context means a tab within the installed PWA window.
+
+### Try it out!
+
+Read the [explainer](https://github.com/WICG/sw-launch/blob/main/launch_handler.md) and [web.dev article](https://web.dev/launch-handler/) to learn more about how to implement the Launch Handler API. To see the API in action, check out the [demo](https://launch-handler.glitch.me/). If you want to start implementing the API in your own app, you can try it out by turning on the `#enable-desktop-pwas-launch-handler` flag in `chrome://flags` or sign up for the [origin trial](https://developer.chrome.com/origintrials/#/view_trial/2978005253598740481).
+
+## Link Capturing
+
+As was previously mentioned, one of the more common ways for your PWA to be launched is from a link. Chrome OS 98 introduces Link Capturing as a user preference which allows users to customize how to open supported in-scope links: in the Chrome browser or in the installed PWA.
+
+Users will get visual cues from Chrome browser to opt-in to link capturing. The current UI and UX may change in the future as the team continues to work on improvements to make the feature more discoverable to users.
+
+By default, links are set to open in the Chrome browser. There are two ways for a user to change their preference:
+
+- In the browser, a user can navigate to an in-scope link and in the omnibox select the “Open with” icon. This will show an intent picker asking them to choose which app to open the link with and a checkbox to remember their choice. If your PWA is installed, it will be available as an option.
+
+![Screen capture of link capturing preferences in the "Open with" intent picker](ix://posts/customize-pwa-window-launch/link_capturing_open_with.gif)
+
+- In the Chrome OS Settings app, a user can go to the “Apps” menu, select “Manage your apps” and find the app they’re interested in (your PWA). Under “Opening supported links”, there are radio button options for opening links in the app or in the Chrome browser.
+
+![Screen capture of link capturing preferences in the settings app](ix://posts/customize-pwa-window-launch/link_capturing_settings_menu.gif)
```

Comment:
```suggestion
By default, links are set to open in the Chrome browser. There are two ways for a user to change their preference, through either the omnibox or system settings.

### Setting preferences through the omnibox

In the browser, a user can navigate to an in-scope link and in the omnibox select the “Open with” icon. This will show an intent picker asking them to choose which app to open the link with and a checkbox to remember their choice. If your PWA is installed, it will be available as an option.

![Screen capture of link capturing preferences in the "Open with" intent picker](ix://posts/customize-pwa-window-launch/link_capturing_open_with.gif)

### Setting preferences through system settings

In the Chrome OS Settings app, a user can go to the “Apps” menu, select “Manage your apps” and find the app they’re interested in (your PWA). Under “Opening supported links”, there are radio button options for opening links in the app or in the Chrome browser.

![Screen capture of link capturing preferences in the settings app](ix://posts/customize-pwa-window-launch/link_capturing_settings_menu.gif)
```

Looking at how this renders, I think making these sub-sections will make this read a little better